From 546aa941d2fd42f69b3e2bd6625d65c145a5e265 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Sat, 16 Sep 2023 22:28:27 +0200
Subject: [PATCH] meson.build: do not enable pidfd features on native glib
 builds

We still use host distros like alma 8 with kernels older than 5.4,
where these features are not implemented.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index a907736..53cffb5 100644
--- a/meson.build
+++ b/meson.build
@@ -1019,7 +1019,8 @@ if cc.links('''#include <sys/syscall.h>
                  waitid (P_PIDFD, 0, &child_info, WEXITED | WNOHANG);
                  return 0;
                }''', name : 'pidfd_open(2) system call')
-  glib_conf.set('HAVE_PIDFD', 1)
+  #requires kernel 5.4+
+  #glib_conf.set('HAVE_PIDFD', 1)
 endif
 
 # Check for __uint128_t (gcc) by checking for 128-bit division
